src/Docfx.Dotnet/SymbolFormatter.cs

```diff
@@ -138,7 +138,9 @@ LinkItem ToLinkItem(SymbolDisplayPart part)
             {
                 Name = overload ? VisitorHelper.GetOverloadId(symbol) : VisitorHelper.GetId(symbol),
                 DisplayName = part.ToString(),
-                Href = SymbolUrlResolver.GetSymbolUrl(symbol, compilation, memberLayout, urlKind, allAssemblies),
+                Href = filter.IncludeApi(symbol)
+                        ? SymbolUrlResolver.GetSymbolUrl(symbol, compilation, memberLayout, urlKind, allAssemblies)
```

Comment:
Should we add the `SymbolFilter` to the `GetSymbolUrl` method to ensure it’s always included?

Reply:
I've moved `SymbolFilter` check logics to `GetSymbolUrl` by db035fe